pymatgen/io/qchem/sets.py

```diff
@@ -521,18 +667,47 @@ def __init__(
     ):
         """
         Args:
-            molecule ():
-            dft_rung ():
-            basis_set ():
-            pcm_dielectric ():
-            smd_solvent ():
-            custom_smd ():
-            opt_variables ():
-            scan_variables ():
-            scf_algorithm ():
-            max_scf_cycles ():
-            plot_cubes ():
-            overwrite_inputs ():
+            molecule (Pymatgen Molecule object)
+            opt_variables (str or list):
+            scan_variables (str or list): 
```

Comment:
@espottesmith can you give me an example of how these variables should be populated for a pes_scan job?